ansible/roles/ssc/templates/start.sh

```diff
@@ -12,6 +18,57 @@ for VAR in $REQUIRED_VARS; do
   fi
 done
 
+# Check if we should start from state sync
+ShouldStartFromStateSync() {
+  if [ -z "$SYNC_RPC" ]; then
+    return 1
+  fi
+  count=$(echo "$SYNC_RPC" | tr ',' '\n' | wc -l)
+  if [ "$count" -ge 2 ]; then
+    return 0
+  else
+    return 1
+  fi
+}
+
+# Configure state sync
+ConfigureStartFromStateSync() {
+  echo "Configuring state sync..."
+  RPC_SERVER=$(echo "$SYNC_RPC" | awk -F"," '{gsub("tcp","http",$1);print $1}')
+  CURRENT_BLOCK=$(curl -s "$RPC_SERVER/status" | jq -r '.result.sync_info.latest_block_height' 2>/dev/null)
```

Comment:
PROBLEM: the ssc container doesn't have `jq` installed. A bit ugly but we can use this:
```
CURRENT_BLOCK=$(
  curl -s "$RPC_SERVER/status" \
    | grep -o '"latest_block_height":"[^"]*"' \
    | sed 's/.*:"//;s/"$//'
)
```